wait/host_port.go

```diff
@@ -130,31 +130,33 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 
 		select {
 		case <-ctx.Done():
-			return fmt.Errorf("%w: %w", ctx.Err(), err)
+			return fmt.Errorf("mapped port: retries: %d port: %s last err: %w ctx err: %w", i, port, ctx.Err(), err)
```

Comment:
Separate items in the message
```suggestion
			return fmt.Errorf("mapped port: retries: %d, port: %s, last err: %w, ctx err: %w", i, port, ctx.Err(), err)
```

Reply:
I used to do that but noticed it caused confusion when the elements themselves included commas, so found this was was more readable, thoughts?

Reply:
Do you find any other separation token that could work, like dashes `-` or semicolon `;`?

Reply:
I didn't I just stuck with space as the "thing: <value>" typically gave enough clarity in practice.
```
mapped port: retries: 1 port: "" last err: port not found ctx err: context deadline exceeded
```

Reply:
If you want to swap to commas for this, it will work as the messages are unlikely to included them.

Reply:
Lets not worry about, commas it is 🤣